<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWindow</class>
 <widget class="QMainWindow" name="ClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client inactive</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="MessageLabel">
        <property name="text">
         <string>Message text:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MessageEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="SendToAllButton">
        <property name="text">
         <string>Send to all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SendPrivateButton">
        <property name="text">
         <string>Send private</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="PublicChatBrowser"/>
      <widget class="QTextBrowser" name="PrivateChatBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLaunch_another_session"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart_session">
   <property name="text">
    <string>Start session</string>
   </property>
  </action>
  <action name="actionLaunch_another_session">
   <property name="text">
    <string>Launch another session</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
